tasks/install-source.yml

```diff
@@ -50,11 +50,21 @@
     mode: 0600
     checksum: "{{ redis_checksum | default(omit) }}"
 
+# In some cases, unarchive will change permissions of destination folder,
+# to prevent permissions issues with directories that might be shared, like /tmp
+# we will create a direcoty specific to this role to extract files to
```

Comment:
The comment is not correct.
The `unarchive` must change the permissions of the destination folder when `mode` parameter is supplied. 
But it behaves differently with `tar.gz` and `zip` archives.
When extracting `tar.gz`, it doesn't touch the destination folder's permissions. 
While in case of extracting `zip` into something like `/tmp/` with `mode: 755`, it overrides the permissions of `/tmp` to 755.

Reply:
Understood,  I will update the comment